<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">

 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">

  <property name="geometry">

   <rect>

    <x>0</x>

    <y>0</y>

    <width>640</width>

    <height>480</height>

   </rect>

  </property>

  <property name="windowTitle">

   <string>Drone Telemetry Simulator</string>

  </property>

  <property name="styleSheet">

   <string notr="true">QWidget#centralwidget {

    background-color: #000000;

    color: #ffffff;

}



QLabel, QGroupBox, QCheckBox, QComboBox, QPushButton {

    color: #ffffff;

}



QGroupBox {

    border: 1px solid #ffffff;

    margin-top: 8px;

}



QGroupBox::title {

    subcontrol-origin: margin;

    subcontrol-position: top left;

    padding: 2px 5px;

}



QPlainTextEdit {

    background-color: #111111;

    color: #ffffff;

    border: 1px solid #555555;

}





QPushButton {

    background-color: #222222;

    border: 1px solid #555555;

    padding: 4px 10px;

    border-radius : 4px;

}



QPushButton:hover {

    background-color: #333333;

}



QPushButton:pressed {

    background-color: #444444;

}





QComboBox {

    background-color: #222222;

    color: #ffffff;

    border: 1px solid #555555;

}



QComboBox QAbstractItemView {

    background-color: #222222;

    color: #ffffff;

    selection-background-color: #444444;

}



QCheckBox {

    spacing: 5px;

}

</string>

  </property>

  <property name="documentMode">

   <bool>false</bool>

  </property>

  <widget class="QWidget" name="centralwidget">

   <layout class="QVBoxLayout" name="mainLayout">

    <item>

     <layout class="QGridLayout" name="controlsTop">

      <item row="3" column="1">

       <widget class="QLabel" name="labelMovement">

        <property name="text">

         <string>Movement Type:</string>

        </property>

       </widget>

      </item>

      <item row="0" column="0">

       <widget class="QPushButton" name="btnStart">

        <property name="text">

         <string>Start</string>

        </property>

       </widget>

      </item>

      <item row="0" column="3">

       <widget class="QPushButton" name="btnStop">

        <property name="text">

         <string>Stop</string>

        </property>

       </widget>

      </item>

      <item row="3" column="2">

       <widget class="QComboBox" name="comboStrategy"/>

      </item>

      <item row="2" column="1">

       <widget class="QCheckBox" name="btnFailure">

        <property name="text">

         <string>Simulate Failure</string>

        </property>

       </widget>

      </item>

      <item row="0" column="1" colspan="2">

       <widget class="QLabel" name="label">

        <property name="text">

         <string>DRONE TELEMETRY SIMULATOR</string>

        </property>

        <property name="alignment">

         <set>Qt::AlignmentFlag::AlignCenter</set>

        </property>

       </widget>

      </item>

     </layout>

    </item>

    <item>

     <widget class="QGroupBox" name="groupTelemetry">

      <property name="title">

       <string>Telemetry</string>

      </property>

      <layout class="QGridLayout" name="telemetryGrid">

       <item row="3" column="2">

        <widget class="QLabel" name="labelGps">

         <property name="text">

          <string>GPS Fix:</string>

         </property>

        </widget>

       </item>

       <item row="2" column="3">

        <widget class="QLabel" name="lblBattery">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="1" column="3">

        <widget class="QLabel" name="lblSpeed">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="0" column="3">

        <widget class="QLabel" name="lblHeading">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="0" column="2">

        <widget class="QLabel" name="labelHeading">

         <property name="text">

          <string>Heading:</string>

         </property>

        </widget>

       </item>

       <item row="1" column="2">

        <widget class="QLabel" name="labelSpeed">

         <property name="text">

          <string>Speed (m/s):</string>

         </property>

        </widget>

       </item>

       <item row="3" column="3">

        <widget class="QLabel" name="lblGps">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="1" column="1">

        <widget class="QLabel" name="lblLat">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="1" column="0">

        <widget class="QLabel" name="labelLat">

         <property name="text">

          <string>Latitude:</string>

         </property>

        </widget>

       </item>

       <item row="0" column="1">

        <widget class="QLabel" name="lblDroneId">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="2" column="2">

        <widget class="QLabel" name="labelBattery">

         <property name="text">

          <string>Battery:</string>

         </property>

        </widget>

       </item>

       <item row="2" column="0">

        <widget class="QLabel" name="labelLon">

         <property name="text">

          <string>Longitude:</string>

         </property>

        </widget>

       </item>

       <item row="3" column="0">

        <widget class="QLabel" name="labelAlt">

         <property name="text">

          <string>Altitude:</string>

         </property>

        </widget>

       </item>

       <item row="0" column="0">

        <widget class="QLabel" name="labelDroneId">

         <property name="text">

          <string>Drone ID:</string>

         </property>

        </widget>

       </item>

       <item row="2" column="1">

        <widget class="QLabel" name="lblLon">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

       <item row="3" column="1">

        <widget class="QLabel" name="lblAlt">

         <property name="text">

          <string>-</string>

         </property>

        </widget>

       </item>

      </layout>

     </widget>

    </item>

    <item>

     <widget class="QPlainTextEdit" name="logView">

      <property name="readOnly">

       <bool>true</bool>

      </property>

     </widget>

    </item>

   </layout>

  </widget>

 </widget>

 <resources/>

 <connections/>

</ui>

